src/pagination/pagination.tsx

```diff
@@ -91,9 +149,58 @@ const Component = (
 
         if (ellipsisStart || ellipsisEnd) {
             return (
-                <EllipsisItem key={pageIndex} focusHighlight={false}>
-                    <EllipsisHorizontalIcon />
-                </EllipsisItem>
+                <EllipsisContainer
```

Comment:
the `:focus` pseudo class doesn't work as div isn't an interactive element

instead of using two separate buttons, you might want to conditionally render `EllipsisHorizontalIcon` or `Chevron2LeftIcon` based on hover state. use `onMouseEnter` and `onMouseLeave` to detect mouse hover events